docs/reference/cluster/reroute.asciidoc

```diff
@@ -10,7 +10,7 @@ Changes the allocation of shards in a cluster.
 [[cluster-reroute-api-request]]
 ==== {api-request-title}
 
-`POST /_cluster/reroute`
+`POST /_cluster/reroute?metric=none`
```

Comment:
We also need to update the docs for the `?metric` option below, describing the `=none` option and also saying that the other options should be avoided.